### ISSUE TITLE
Question: members group

### ISSUE THREAD

**Issue by bartrylant** [designbyfront/LDAP-Authentication-for-ExpressionEngine#16]
Does the extension remember which group the user has been assigned to (if you assigned him to another than the default member group) or is he assigned to the default group again every time the credentials are compared with the LDAP server information?

**Comment by lightfuze** [comment]
Yes, once a member has been assigned to a group in EE, they will remain a member of that group.
